DOCUMENTATION.md

```diff
@@ -1,10 +1,60 @@
-Welcome to the Balena SDK documentation.
+Welcome to the Balena SDK documentation. This document aims to describe all the functions supported by the SDK, as well as showing examples of their expected usage.
+
+## Installation
+
+Install the balena SDK by running:
+
+```sh
+$ npm install --save balena-sdk
+```
```

Comment:
Would be nice to have an `### npm` & `### CDN` subsection in the `## Installation`, to mention unpkg like in the README:
> The balena SDK also provides a bundled single file for browsers, which allows you to include a single file with all dependencies included, available as [balena-browser.min.js](https://unpkg.com/balena-sdk/es2015/balena-browser.min.js) (or [balena-browser.js](https://unpkg.com/balena-sdk/es2015/balena-browser.js) if you'd like the much larger unminified version). This uses the [UMD format](https://github.com/umdjs/umd), and will register itself as either a CommonJS or AMD module called `balena-sdk` if possible, or create a `balenaSdk` global if not.